purchase_order_secondary_unit/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2018 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    'name': 'Purchase Order Secondary Unit',
+    'summary': 'Purchase product in a secondary unit',
+    'version': '11.0.1.0.0',
+    'development_status': 'Beta',
+    'category': 'Purchase',
+    'website': 'https://github.com/OCA/purchase-workflow',
+    'author': 'Tecnativa, Odoo Community Association (OCA)',
+    'license': 'AGPL-3',
+    'application': False,
+    'installable': True,
+    'auto_install': True,
```

Comment:
IMO auto_install should be False for this module.

Reply:
If you have installed purchase_secondary_unit and purchase, it's logic that this should be auto-install, isn't it?

Reply:
I misread the module name I thought it was purchase_secondary_unit, product_secondary_unit makes more sense.